### PR TITLE
Confirmation modal

### PR DESCRIPTION
**Pull request by sdevalapurkar-bcgov**
# Description

This PR includes the following proposed change(s):

- creation of confirmation modal
- Jira: https://justice.gov.bc.ca/jira/browse/FLA-143

Capture

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Local, storybook and jest

### CODE REVIEW

**Comment by akroon3r** [comment]
Shared component tests failed:   

 Header
      ✕ Default (7ms)
      ✕ Mobile (5ms)

File: src/frontend/shared-components/src/storyshots.test.js [hunk 1]
```diff
@@ -2,4 +2,5 @@ import initStoryshots from "@storybook/addon-storyshots";
 
 initStoryshots({
   /* configuration options */
+  storyKindRegex: /^((?!.*?ConfirmationPopup).)*$/
```
Comment (adodge-bcg) [inline src/frontend/shared-components/src/storyshots.test.js:5]: What is this doing?

File: src/frontend/shared-components/src/loader/Loader.stories.js [hunk 2]
```diff
@@ -7,12 +7,6 @@ export default {
   component: Loader
 };
 
-export const Default = () => (
-  <div style={{ background: "grey" }}>
-    <Loader />
-  </div>
-);
```
Comment (adodge-bcg) [inline src/frontend/shared-components/src/loader/Loader.stories.js:14]: Reason for removing the default?

Comment (sdevalapurkar-bcgov) [reply src/frontend/shared-components/src/loader/Loader.stories.js:14]: yeah dont need the grey background, the default is `Loader` 😄

File: src/frontend/shared-components/src/storyshots.test.js (hunk 1 above)
Comment (sdevalapurkar-bcgov) [reply src/frontend/shared-components/src/storyshots.test.js]: no there isnt it is outdated now :)

File: src/frontend/shared-components/src/confirmation-popup/ConfirmationPopup.stories.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  show: true,
+  handleShow: () => {},
+  handleClose: () => {},
+  title: "Cancel process?",
+  body1: "Your process will end.",
+  body2: "You will be redirected back to the original application."
+};
+const mainButton = { mainLabel: "main label", mainStyling: "normal-blue btn" };
+const confirmButton = {
+  confirmLabel: "Yes, cancel my process please",
```
Comment (akroon3r) [inline src/frontend/shared-components/src/confirmation-popup/ConfirmationPopup.stories.js:20]: missing period? not a blocker, might be missing context.

File: src/frontend/shared-components/src/confirmation-popup/ConfirmationPopup.stories.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  confirmStyling: "normal-blue btn consistent-width"
+};
+const cancelButton = {
+  cancelLabel: "No, resume my process please",
```
Comment (akroon3r) [inline src/frontend/shared-components/src/confirmation-popup/ConfirmationPopup.stories.js:24]: same

File: src/frontend/shared-components/src/confirmation-popup/ConfirmationPopup.stories.js (hunk 3 above)
Comment (sdevalapurkar-bcgov) [reply src/frontend/shared-components/src/confirmation-popup/ConfirmationPopup.stories.js:20]: this is just in the story so not important in terms of actual wording, so going to leave it :)

**Pull request merged**